server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraMailboxModule.java

```diff
@@ -85,7 +85,6 @@
 
     @Override
     protected void configure() {
-        install(new DefaultEventModule());
```

Comment:
what is the need to move that one?

Reply:
The EventBus implementation seems othogonal to the Cassandra mailbox implementation choice.